tests/integration/targets/vmware_content_library_item_info/vars.yml

```diff
@@ -0,0 +1,6 @@
+vcenter_hostname: "127.0.0.1"
```

Comment:
I don't see test for simulator. Will you add it later?

Reply:
I cannot figure out how to mock the rest api for this module, so I dont think I can include the simulator tests at this time

Reply:
In this case better delete `vars.yml` and remove it from `run.yml`